service/handlers_test.go

```diff
@@ -48,8 +49,36 @@ func (d mockDB) CreateProjectEntry(ctx context.Context, pe db.ProjectEntry) erro
 	return nil
 }
 
+func (d mockDB) ListTokenEntries(ctx context.Context, project string) ([]db.TokenEntry, error) {
+	if project == projectDoesNotExist {
+		return []db.TokenEntry{}, upper.ErrNoMoreRows
+	}
+
+	if project == "projectnotokens" {
+		return []db.TokenEntry{}, nil
+	}
+
+	return []db.TokenEntry{
+		{
```

Comment:
@bkono this is very hardcode-y. Is it worth the time and effort to do some refactoring?